.travis.yml

```diff
@@ -13,12 +13,19 @@ script:
 before_deploy:
   - node ./travis/bumpversion.js
 deploy:
-  provider: npm
-  # Do not throw away the updated package.json we generated in `before_deploy`:
-  skip_cleanup: true
-  email: '$NPM_EMAIL'
-  api_key: '$NPM_TOKEN'
-  # Note: do not deploy on pull request, because $TRAVIS_BRANCH will be the target branch.
-  tag: '$TRAVIS_BRANCH'
-  on:
-    all_branches: true
+  - provider: npm
+    # Do not throw away the updated package.json we generated in `before_deploy`:
+    skip_cleanup: true
+    email: '$NPM_EMAIL'
+    api_key: '$NPM_TOKEN'
+    # Note: do not deploy on pull request, because $TRAVIS_BRANCH will be the target branch.
+    tag: '$TRAVIS_BRANCH'
+    on:
+      all_branches: true
+  - provider: pages
```

Comment:
@megoth I added gh-pages deployment using travis, since that is what we have in place anyway. I think someone with travis access (you?) needs to add a `$GITHUB_TOKEN`

Reply:
You should have access, as you are administrator in this repo.

Reply:
You don't have access to https://travis-ci.org/github/solid/solid-ui ? I think you should, as you are administrator in this repo.

Reply:
You are right, wasn't aware of that